### ISSUE TITLE
Do labels exist for the Spacenet Testing datasets?

### ISSUE THREAD

**Issue by 00krishna** [NVIDIA/DIGITS#2029]
Say, I have been working with the Spacenet building dataset and found an odd artifact, that I was hoping someone could confirm, perhaps @jbarker-nvidia . 

So there is a directory in each Spacenet Training dataset that provides the labels for that dataset. So there is a set of labels for the Las Vegas Training data, the Paris Training data, etc. However I did not find any existing labels for the Space public Testing datasets for any of the nations. I was thinking that the Testing datasets would be used for validation, but without labels we can't do any validation or compare our results to say the results from the DIGITS blog. Did other people find the same thing?

I suppose I can create my own testing dataset  by selecting out a few Training images. I figure that the Nvidia folks who wrote the blog posts on Detectnet and Spacenet must have had to do a similar thing. If that is the case, could any of the nvidia folks explain how they created their Testing data or conducted their testing. I was just trying to benchmark my results against theirs, so any info would help.

**Comment by jbarker-nvidia** [comment]
@00krishna I would suggest asking this on the SpaceNet github repo: https://github.com/SpaceNetChallenge

I don't know the answer, but I would guess that the test set was the data that was used to judge performance in the SpaceNet competition - in which case the labels wouldn't have been made public.  For the blog posts we wrote we just selected a random ~20% of chips from the training data to use as a validation set.

Please close this issue if you're able to pick up the question on the SpaceNet repo.  Thanks!

**Comment by 00krishna** [comment]
Okay great, that makes sense. I was just surprised when I noticed this--especially since the competition is over. I can close the issue.
